tools/update_chrome_revision.mjs

```diff
@@ -107,42 +105,59 @@ async function updateDevToolsProtocolVersion(revision) {
   );
 }
 
+async function saveVersionData(data) {
+  await writeFile('./versions.json', JSON.stringify(data, null, 2));
```

Comment:
This will make sure prettier does not complain for formatting.
```suggestion
  await writeFile('./versions.json', JSON.stringify(data, null, 2));
  touchedFiles.push('./versions.json');
```